packages/react-router-dom/server.tsx

```diff
@@ -98,6 +97,7 @@ export function StaticRouterProvider({
     router,
     navigator: getStatelessNavigator(),
     static: true,
```

Comment:
Can we get rid of the static prop in favor of the existence of the static context?

Reply:
Not quite yet - we still need `static` for non-data-routers doing SSR